app/org/sagebionetworks/bridge/json/BridgeObjectMapper.java

```diff
@@ -55,6 +55,7 @@ public static final BridgeObjectMapper get() {
         return INSTANCE;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Minor: Instead of suppressing the deprecation warning, it would be better to change the call from setFilters() to setFilterProvider(), which does the exact same thing and is the intended replacement for setFilters()